glossary.md

```diff
@@ -272,6 +272,8 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | multi-class classification   | phân loại đa lớp              | [https://git.io/Jvoj0](https://git.io/Jvoj0) |
 | multinominal distribution    | phân phối đa thức             | [https://git.io/JvohQ](https://git.io/JvohQ) |
 | multitask learning           | học đa nhiệm                  | [https://git.io/JvohQ](https://git.io/JvohQ) |
+| multi-head attention           | tầng tập trung đa đầu                  |  |
+| masked multi-head attention    | tầng tập trung đa đầu có mặt nạ        |  |
```

Comment:
có từ layer đâu nhỉ
```suggestion
| multi-head attention           | tập trung đa đầu                  |  |
| masked multi-head attention    | tập trung đa đầu có mặt nạ        |  |
```

Reply:
Mình thấy nên thêm từ `layer` vào phần tiếng anh. Cụm `tập trung đa đầu` đứng không không có nghĩa lắm

Reply:
mình đang flexible cho mấy từ kiểu `multi-head attention mechanism` nhưng vừa search thì ko có :))
```suggestion
| multi-head attention layer          | tầng tập trung đa đầu                  |  |
| masked multi-head attention layer    | tầng tập trung đa đầu có mặt nạ        |  |
```

Reply:
Mấy cụm này mình lấy trong ảnh:
![transformer.svg](https://raw.githubusercontent.com/aivivn/d2l-vn/master/img/transformer.svg)

Trong này không có layer nhưng mình nghĩ có thể dịch chúng như `tầng` hay `kiến trúc` hay `cơ chế` xxx gì đấy.

Reply:
như trong hình này thì imply là tầng rồi, mình thấy để cả từ `layer` trong glossary như trên ok

Reply:
Theo mình hiểu nguyên cụm multi-head attention có nghĩa là concat nhiều attention lại với nhau nhằm mục đích giúp model có thể nắm bắt thông tin từ nhiều trường không gian đại diện cho từ hơn.
Cụ thể hơn như trong câu: “The animal did not cross the street because it was tired.” Nếu có multi-head attention (giả sử là có 2 heads)
khi truy vấn từ “it”, rất có khả năng nó sẽ chú ý vào 2 từ: “animal” nhờ đầu tập trung thứ nhất tập trung vào từ mà từ it đại diện và “tired” nhờ đầu tập trung thứ hai tập trung vào tính từ bổ nghĩa cho từ đó.

Mình Ko rành NLP lắm, mình chỉ hiểu đại khái mỗi “head” như là một mục tiêu chú ý trong những trường không gian (subspaces) cho embedding vector của từ.

Theo thiển ý của mình khi dịch, “đầu” Ko phải là “cái đầu” mà là một “đầu mối/mục tiêu” để tập trung.

Reply:
Có một hình ảnh gần nhất với cách dùng từ `head` của tác giả trong trường hợp này là `read / write head -- đầu đọc / đầu ghi` trong ổ cứng.
Mỗi attention-head sẽ tập trung vào một phần của input và đọc từ các ô đó (hoặc ghi vào các ô đó).

Một từ luôn có nhiều cách dịch, nếu nhìn từ góc đồ thị tính toán thì dịch là `nhánh` cũng đúng, nhưng mà như vậy trong DL chỉ cần không phải sequential thì  hầu như cái gì dịch là `nhánh` cũng đúng hết. 

Mới đọc mình cũng thấy `đa đầu` nghe hơi kì nhưng bây giờ thấy bình thường, nhất là khi đọc trong ngữ cảnh. Nếu chọn thì mình luôn chọn cách dịch gần với hàm ý nghĩa gốc nhất có thể.